### DISCUSSION TITLE
Welcome to BRENDApyrser Discussions!

### DISCUSSION THREAD

**Discussion by Robaina** [Robaina/BRENDApyrser#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gallevin** [comment]
Hi, is there way to retrieve the KI value and MW of a species enzyme (based on his ec number) ? 


for example  - I was trying to add KI value for 4.1.1.39 as follow: 


Enzyme_EC  = 4.1.1.39
r = brenda.reactions.get_by_id(Enzyme_EC)
compound = 'CO2'
KI = r.KIvalues.filter_by_compound(compound).get_values()
plt.hist(KI)
plt.xlabel('KI (mM)')
plt.title(f'{r.name} ({compound}, median_kI = {np.median((KI))})')
plt.show()
